extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -538,7 +579,14 @@ export class AmpAnalytics extends AMP.BaseElement {
             request, undefined, whiteList);
       })
       .then(request => {
-        this.sendRequest_(request, trigger);
+        if (this.config_['requestsFrameUrl']) {
+          const requestsFrameKey = this.buildRequestsFrameUrlWithDataHash(
+              this.config_['requestsFrameUrl'], this.config_['dataHash']);
+          AmpAnalytics.requestsFrames.send(requestsFrameKey,
+            request);
```

Comment:
This is another case where extracting requestsFrameKey is not strictly necessary, but looks better to my eye, as to not do so would be very cluttered. Please let me know if anyone disagrees.